tennis-scheduler-web-app/src/App.js

```diff
@@ -1,24 +1,26 @@
-import logo from './logo.svg';
 import './App.css';
+import React, { useState } from "react"
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import { Login } from './components/Login';
+import { Registration } from './components/Registration';
+import { StartPage } from './components/StartPage';
+import { NavbarStart } from './components/Navbar';
+import { HomePage } from './components/HomePage';
+
 
 function App() {
+
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-         Hello from tennis scheduler
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
```

Comment:
No need for fragment + div if you don't consider adding sibling element to that div.